### PR TITLE
Chronic Kidney Disease Analysis

### LINKED ISSUE #12: Add a new project: Kidney disease prediction
I would like to add a new project for predicting kidney diseases. 
Using the dataset : https://www.kaggle.com/mansoordaku/ckdisease  
I am a participant of GSSOC'21

### PR DESCRIPTION
**Pull request by sudipg4112001**
### Related issue
Fixes #12 

Notebook and dataset added

### CODE REVIEW

**Review by utkarsh0702 — CHANGES_REQUESTED** [review]
@sudipg4112001 The visualizations are great 👍.  Some things to improve:
1) Please don't use mean methord to interpolate missing data, use pandas provided other interpolation techniques.
2) Please specify the models validation accuracy, you have provided training accuracy and even I feel like the model overfitted.
3) Attach the model with highest accuracy.

**Comment by utkarsh0702** [comment]
@sudipg4112001 Please provide an update regarding the pr.

**Comment by kritikaparmar-programmer** [comment]
@sudipg4112001 Please provide the updates regarding the changes asked, if you're facing any kind of problem kindly contact mentors.

**Comment by shubhigupta991** [comment]
@sudipg4112001 Updates?
